source/_integrations/omnilogic.markdown

```diff
@@ -22,12 +25,44 @@ There is currently support for the following device types within Home Assistant:
 
 - ***Sensor*** - Air Temperature, Water Temperature, Variable Pump Speed, Chlorinator Setting, Salt Level, pH, and ORP
 - ***Switch*** - All relays, pumps (single, dual, variable speed), and relay-based lights.
+- ***Light*** - Colorlogic Lights (V1 and V2).
 
 {% include integrations/config_flow.md %}
 
+## Sensor Platform Options
+
+If you have pH sensors in your Omnilogic setup, you can add an offset to correct reporting from the sensor in the integration configuration.
+
+## Switch Platform
+
+The switch platform contains a custom service to allow you to set the speed on variable speed pumps.
+
+To set pump speed, call the `omnilogic.set_pump_speed` service as following:
```

Comment:
Why make this a service? Instead, you could be providing a number entity?

Reply:
I will look at the number entity option - I don't believe this was available when we originally submitted the PR.